package.json

```diff
@@ -48,6 +48,7 @@
     "check:dependencies": "yarn workspaces run check:dependencies",
     "check:license": "node scripts/check-licenses.js",
     "check:vulnerability": "yarn audit-ci -h",
+    "check:test-service": "yarn generate:test-services:v2 && yarn generate:test-services:v4 && yarn ts-node scripts/check-test-service.ts",
```

Comment:
Minor: Just a matter of taste. You could add the generation of the test services also to the typescript file.